packages/backend-core/tests/utilities/DBTestConfiguration.ts

```diff
@@ -12,6 +12,10 @@ class DBTestConfiguration {
     this.tenantId = structures.tenant.id()
   }
 
+  reset() {
```

Comment:
Is this reset? Or `useNewDb`?

Reply:
Probably the most correct term would be `newTenant` - I'll update

Reply:
https://github.com/Budibase/budibase/pull/9760/commits/6144416d549c2ac773f5511dcd82b46fc21c4b6a